docs/markdown/Reference-tables.md

```diff
@@ -85,6 +88,7 @@ set in the cross file.
 | sparc64             | SPARC v9 processor       |
 | wasm32              | 32 bit Webassembly       |
 | wasm64              | 64 bit Webassembly       |
+| dsPIC33E            | 16 bit Microchip dsPIC   |
```

Comment:
Same thing here as above.

Reply:
and I can make this `dsPIC`

Reply:
All cpu family values should be in all lower case. Another question is how does this relate to other PIC processors? Are they nominally the same as others? If yes, how should they be named? Plain pic? pic8/pic16/pic32?

Reply:
Yeah I didn't want to fill in all of them here as it would be alot
so I've just added the xc16 compiler here

xc16 (16bit compiler)
-dsPIC
-pic16 and pic24 and a few others I think

xc8 (8bit compiler)
-pic8

xc32 (32bit compiler)
-pic32

Reply:
in the cross file that exact CPU has to be defined  `'-mcpu=33EP64MC203'`

Reply:
Let's just put `dspic` for now, the others can be added if and when people add support for those.